packages/attestation-service/README.md

```diff
@@ -0,0 +1,33 @@
+# Attestation Service
+
+This is the attestation service that validators run on the Celo network to provide attestations of phone numbers to accounts.
```

Comment:
Nit, consider:
`A service run by validators on the Celo network to send SMS messages, enabling attestations of user phone numbers.`